src/components/TimeseriesDataExport/stories/basic.story.mdx

```diff
@@ -26,7 +26,7 @@ import 'antd/dist/antd.css';
 import React from 'react';
 import { TimeseriesDataExport } from '@cognite/gearbox';
 ```
-we use small functional component for demostration.
+We use small Functional component to demonstrate basic behavior.
```

Comment:
I meant to remove word `we`

We use small Functional component to demonstrate basic behavior -> Wrapper component to demonstrate basic behavior